src/courtroom.cpp

```diff
@@ -3522,6 +3568,12 @@ void Courtroom::on_music_search_edited(QString p_text)
     }
   }
 }
+void Courtroom::on_music_search_keypr()
+{
+  if (ui_music_search->text() == "" && !ui_music_list->isHidden()) {
+    ui_music_list->collapseAll(); // quick shortcut to collapse the whole list
+  }
```

Comment:
So if you clear the search bar, it collapses everything instead of returning it to the previous state before you started searching? Sounds kind of annoying.

Reply:
the name "keypr" is misleading, this actually only happens when you press enter/return with an empty search bar. this is due to porting this code directly from 2.7 without changing variable names that were misleading in that branch

Reply:
If it's just on pressing Enter on an empty search bar, I think it's fine?
Though it still might end up being annoying, not sure about it.

Reply:
since there isn't really any other reason to be pressing "enter" in the music list, and since it has to be empty, it's unlikely this will happen by accident as far as I'm concerned